airflow/executors/kubernetes_executor.py

```diff
@@ -265,7 +303,10 @@ def run_pod_async(self, pod: V1Pod, **kwargs):
         return resp
 
     def _make_kube_watcher(self) -> KubernetesJobWatcher:
-        resource_version = ResourceVersion().resource_version
+        resource_instance = ResourceVersion(
+            kube_client=self.kube_client, namespace=self.kube_config.kube_namespace
+        )
+        resource_version = resource_instance.resource_version  # pylint: disable=no-member
```

Comment:
```suggestion
        resource_version = ResourceVersion(
            kube_client=self.kube_client, namespace=self.kube_config.kube_namespace
        ).resource_version  # pylint: disable=no-member
```

(Untested, might have linting problems)